### PR TITLE
Multi sass binary

### LINKED ISSUE #22: Build multiple sass files from one rule
I encountered a need to compile multiple sass files in a large angular application. It would be a great feature to support globs of sass files.

I created a new rule `multi_sass_binary` which accomplishes this task.

### PR DESCRIPTION
**Pull request by stoltene2**
Closes #22

### CODE REVIEW

File: sass/test/sass_rule_test.bzl [hunk 1]
```diff
@@ -39,6 +39,21 @@ def _sass_binary_test(package):
...
+            "nested/deep/main.css.map",
+        ],
+        rule = package + "/glob_sass",
+    )
+
+
 def sass_rule_test(package):
     """Issue simple tests on sass rules."""
     _sass_binary_test(package)
+    _multi_sass_binary(package)
```
Comment (perezd) [inline sass/test/sass_rule_test.bzl]: suffic with _test please.

**Comment by stoltene2** [comment]
Good catch. Updated and resolved merge conflict.

**Comment by stoltene2** [comment]
@perezd can you review the change?

File: sass/test/sass_rule_test.bzl (hunk 1 above)
Comment (stoltene2) [reply sass/test/sass_rule_test.bzl]: done

**Comment by perezd** [comment]
Yes, sorry!

**Pull request merged**